tsl/test/sql/recompress_chunk_segmentwise-v2.sql

```diff
@@ -0,0 +1,477 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP DATABASE IF EXISTS recompression_optimization_db;
+CREATE DATABASE recompression_optimization_db;
```

Comment:
sorry if it's obvious but why do we need to create a new database for this test?

Reply:
Right, removed it now.